runtime/Dart/lib/src/error/error.dart

```diff
@@ -4,6 +4,9 @@
  * can be found in the LICENSE.txt file in the project root.
  */
 
+export 'src/console_error_listener_stub.dart'
+    if (dart.library.io) 'src/console_error_listener_io.dart'
+    if (dart.library.html) 'src/console_error_listener_html.dart';
```

Comment:
Instead of making the whole console error listener, I prefer extracting just the `stderr.writeln` method like what you did in other places.

Reply:
Thanks for the suggestion. Updated.